ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -698,8 +700,16 @@ static KsqlRestApplication buildApplication(
     final RoutingFilterFactory routingFilterFactory = initializeRoutingFilterFactory(ksqlConfig,
         heartbeatAgent, lagReportingAgent);
 
+    final Boolean collectMetrics = ksqlConfig.getBoolean(
```

Comment:
nit: can't be `null`, so:

```suggestion
    final boolean collectMetrics = ksqlConfig.getBoolean(
```

Reply:
Removed temp variable